### PR TITLE
#950: Refactored Java Script Options parser

### PR DESCRIPTION
**Pull request by tomuben**
### All Submissions:

* [x] Check if your pull request goes to the correct bash branch (develop or master)?
* [x] Is the title of the Pull Request correct?
* [x] Is the title of the corresponding issue correct?
* [x] Have you updated the changelog?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change?  
* [x] Are you mentioning the issue which this PullRequest fixes ("Fixes...")

### When integrating to Develop branch:

1. [x] Remember to merge with "Squash and Merge"

### When integrating to Main branch:

1. [ ] Remember to merge with "Merge"

### CODE REVIEW

File: .github/workflows/check_bazel_tests.yml [hunk 1]
```diff
@@ -16,10 +16,6 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-
-    - name: Search for duplicated error codes
```
Comment (tkilias) [inline .github/workflows/check_bazel_tests.yml]: Why did you remove it?

Comment (tomuben) [reply .github/workflows/check_bazel_tests.yml]: There is another workflow which runs the same check:
https://github.com/exasol/script-languages-release/blob/refactoring/950_refactor_options_line_parser/.github/workflows/check_error_codes.yml

**Review by tkilias — APPROVED** [review]
